docs/general/how-to-define-semantic-conventions.md

```diff
@@ -0,0 +1,151 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: How To Define New Semantic Conventions
+aliases: [how-to-define-new-semantic-conventions]
+--->
+
+# How to define new semantic conventions
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [Defining new conventions](#defining-new-conventions)
+  - [Best practices](#best-practices)
+    - [Defining attributes](#defining-attributes)
+    - [Defining spans](#defining-spans)
+    - [Defining metrics](#defining-metrics)
+    - [Defining resources](#defining-resources)
+    - [Defining events](#defining-events)
+- [Stabilizing existing conventions](#stabilizing-existing-conventions)
+  - [Migration plan](#migration-plan)
+
+<!-- tocstop -->
+
+This document describes requirements, recommendations, and best practices on how to define conventions
+for the new areas or make substantial changes to the existing ones.
+
+## Defining new conventions
+
+- New conventions MUST have a group of codeowners. See [project management](https://github.com/open-telemetry/community/blob/main/project-management.md) for more details.
+  <!-- TODO: add CI check for CODEOWNERS file (when a new area is added) -->
+- New conventions SHOULD be defined in YAML files. See [YAML Model for Semantic Conventions](/model/README.md) for the details.
+- New conventions SHOULD be defined with `development` stability level.
+- New conventions SHOULD include telemetry signal definitions (spans, metrics, events, resources, profiles) and MAY include new attribute definitions.
+
+### Best practices
+
+#### Defining attributes
+
+Reuse existing attributes when possible. Look through [existing conventions](/docs/attributes-registry/) for similar areas,
+check out [common attributes](/docs/general/attributes.md).
+Semantic conventions authors are encouraged to use attributes from different namespaces.
+
+Introduce new attributes when there is a clear use-case for them. Consider adding
+them if most of the following apply:
+
+- They provide a clear benefit to end users by enhancing telemetry.
+- You plan to use the attribute in spans, metrics, events, resources, or other telemetry signals.
+- The attribute will be utilized in instrumentations.
```

Comment:
```suggestion
- The attribute will be utilized in instrumentations.
```
If the above is true, isn't this already somewhat implicit? (that they are used in spans/metrics/logs etc? 

I think what we have been trying to push when folks want to add new things, is that we'd like to see implementations of them, so maybe we can also say:

> - There is a clear plan on how these attributes will be used by instrumentations

Reply:
> If the above is true, isn't this already somewhat implicit? (that they are used in spans/metrics/logs etc?

I'd like to avoid someone defining conventions without planning to actually implement it. Changed the wording slightly to make it more clear.

Also, some of the attributes are not really part of any specific telemetry signal. E.g. `code`, `thread`, `enduser`, `session` - it's something you stamp on any span/log/etc to enrich it. For those we'd probably not require referencing them on a span/metric/etc, but would ask about instrumentations and some plaintext description of how they are used.

> There is a clear plan on how these attributes will be used by instrumentations

I like this, thanks!